### PR TITLE
Clear filters with escape keystroke

### LINKED ISSUE #415: Keyboard shortcut to clear filters
Would be great to be able to clear filters via a keyboard shortcut, say <kbd>Esc</kbd>.

### PR DESCRIPTION
**Pull request by 3minus1**
Worked on Issue #415 

Applied filters can be cleared with Esc key.

Fixes #415

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -173,7 +173,8 @@ var shortcuts = {
   79:  openCurrentLink,   // o
   191: openModal,         // ?
   190: sync,              // .
-  82:  sync               // r
+  82:  sync,              // r
+  27:  clearFilters
```
Comment (samjacobclift) [inline app/assets/javascripts/application.js]: I guess you need a '// esc' like the above lines

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -279,6 +280,10 @@ function autoSync() {
   hasMarkedRows() || sync()
 }
 
+function clearFilters(){
+  Turbolinks.visit("/");
+}
+
```
Comment (samjacobclift) [inline app/assets/javascripts/application.js]: Just a nit we need a space before the '{'  :] does the 'visit' remove the urls query params too?

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (3minus1) [reply app/assets/javascripts/application.js]: Oh, missed that.

Comment (3minus1) [reply app/assets/javascripts/application.js]: Fixed.

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (3minus1) [reply app/assets/javascripts/application.js]: Yes it does.

Comment (3minus1) [reply app/assets/javascripts/application.js]: And space added 👍

**Comment by andrew** [comment]
Awesome, thanks @3minus1!

**Pull request merged**